MobileCenterAnalytics/MobileCenterAnalytics/Internals/MSAnalyticsInternal.h

```diff
@@ -23,7 +25,7 @@
  * @param pageName  page name.
  * @param properties dictionary of properties.
  */
-+ (void)trackPage:(NSString *)pageName withProperties:(NSDictionary *)properties;
++ (void)trackPage:(NSString *)pageName withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties;
```

Comment:
Don't mind. I just checked previous PR and found that I missed the change in the header file. 😄 
BTW, I was talking about the key/value types for NSDictionary, not nullable.